gossipsub.go

```
@@ -522,6 +529,45 @@ func (gs *GossipSubRouter) Attach(p *PubSub) {
 	}
 }
 
+func (gs *GossipSubRouter) manageAddrBook() error {
```

could we just not return anything?

---

it should, the error is ignored anyway.

---

Can you return nothing and log errors?
These are just swallowed now.